Code/6-wire-version/src/main.cpp

```diff
@@ -21,78 +21,113 @@ void setup() {
 			// int dsp_audio_pin 	= D6 
 			// );
 	//example: bwc.begin(D1, D2, D3, D4, D5, D6, D7);
-  updateMqttTimer.attach(600, sendMQTTsetFlag);     //update mqtt every 10 minutes. Mqtt will also be updated on every state change
-  updateWSTimer.attach(2.0, sendWSsetFlag);         //update webpage every 2 secs plus state changes
-  periodicTimer.attach(60, periodicTimerCallback);  //things to do once in a while
-  LittleFS.begin(); // needs to be loaded here for reading the wifi.json
+
+  // check things every 60 seconds
+  periodicTimer.attach(60, []{ periodicTimerFlag = true; });
+  
+  // update webpage every 2 seconds. (will also be updated on state changes)
+  updateWSTimer.attach(2.0, []{ sendWSFlag = true; });
+
+  // update MQTT every 10 minutes. (will also be updated on state changes)
+  updateMqttTimer.attach(600, []{ sendMQTTFlag = true; });
+
+  // needs to be loaded here for reading the wifi.json
+  LittleFS.begin();
   loadWifi();
   startWiFi();
   startNTP();
   startOTA();
-  startServer();
+  startHttpServer();
   startWebSocket();
   startMQTT();
+
   bwc.print(WiFi.localIP().toString());
-  Serial.println(F("\nEnd of setup()"));
+  Serial.println(F("End of setup()"));
```

Comment:
ESP spits out some data at startup, so the first print we make will not be on a new line. Hence the '\n'. Doesn't matter really, I just used this for debugging. Can be removed.

Reply:
got you, maybe we should add a line break at the very beginning of the setup()

Reply:
Let's say I want to do that (I want but that's not the point) what would be the proper way for me to go? Should I open the modified file in vsc, or wait until after merging this PR? Or third option?

Reply:
i think the git flow is about to create a origin/feature/branchXY and then PR to the dev branch. on changes at the dev branch, all other feature branches have to merge dev into them on merge conflicts. guess in your case you can just push your changes to your dev branch because you don't need anyone reviewing your code (bitcoin miner hax).. :D